cumulus_library/template_sql/column_datatype.sql.jinja

```diff
@@ -4,5 +4,5 @@ SELECT
 FROM information_schema.columns
 WHERE
     table_schema = '{{ schema_name }}'
-    AND table_name = '{{ table_name }}'
-    AND LOWER(column_name) IN ('{{ column_names|join("', '") }}') --noqa: LT05
+    AND table_name = '{{ table_name|lower }}'
```

Comment:
I tried this same line with `LOWER(table_name)` and Athena took a minute and then errored out about `DELTA_LAKE_INVALID_SCHEMA` and `Metadata not found in transaction log...` etc.  :shrug: - removing the lower fixes that oddity and doesn't super matter - we don't make mixed case tables today.